app/views/_includes/application-card.njk

```diff
@@ -26,9 +26,17 @@
           <span data-qa="provider">{{ a.provider }}</span>
           {{a.accreditedBody}}
         </dd>
+            {% if a.personalDetails.givenName == 'Peter'%}
+         <dd class="govuk-hint govuk-!-font-size-16 govuk-!-margin-top-0">
+          {# <span style="color: #f47738">&bull;</span>  #}
+          Updated on 20 July 2023 at 11:46am</dd>
```

Comment:
For these ones, can we add the name so 'Peter Essien updated their application on 20 July...'
This is so providers don't get confused about who updated it.